api/envoy/admin/v3/config_dump.proto

```diff
@@ -32,6 +32,8 @@ message ConfigDump {
   //
   // * ``bootstrap``: :ref:`BootstrapConfigDump <envoy_v3_api_msg_admin.v3.BootstrapConfigDump>`
   // * ``clusters``: :ref:`ClustersConfigDump <envoy_v3_api_msg_admin.v3.ClustersConfigDump>`
+  // * ``ecds_filter_http``: :ref:`EcdsConfigDump <envoy_v3_api_msg_admin.v3.EcdsConfigDump>`
+  // * ``ecds_filter_tcp_listener``: :ref:`EcdsConfigDump <envoy_v3_api_msg_admin.v3.EcdsConfigDump>`
```

Comment:
Should `ecds_filter_upstream_http` and `ecds_filter_udp_listener` also be added here, or is it because they are not plumbed at the moment?

Reply:
That's a good question.  ecds_filter_udp_listener is not plumbed yet so we should leave it out.

ecds_filter_upstream_http leveraged most of the ecds_filter_http code, so theoretically it should work automatically.  However, I don't see any Upstream HTTP filter related test code thus guess it is WIP.  Let's leave it out until the feature is ready and related tests are in place.